README.md

```diff
@@ -21,9 +21,9 @@ If you are running into an issue and cannot find any information in the troubles
 
 ## Cromwell on Azure 
 
-[Cromwell](https://cromwell.readthedocs.io/en/stable/) is a workflow management system for scientific workflows, orchestrating the computing tasks needed for genomics analysis. Originally developed by the [Broad Institute](https://github.com/broadinstitute/cromwell), Cromwell is also used in the GATK Best Practices genome analysis pipeline. Cromwell supports running scripts at various scales, including your local machine, a local computing cluster, and on the cloud. Cromwell workflows are written using the open source [WDL](https://github.com/openwdl/wdl) or the [CWL](https://www.commonwl.org/) scripting languages. To see examples of WDL scripts - see this ['Learn WDL'](https://github.com/openwdl/learn-wdl) Repo on Github<br/>
+[Cromwell](https://cromwell.readthedocs.io/en/stable/) is a workflow management system for scientific workflows, orchestrating the computing tasks needed for genomics analysis. Originally developed by the [Broad Institute](https://github.com/broadinstitute/cromwell), Cromwell is also used in the GATK Best Practices genome analysis pipeline. Cromwell supports running scripts at various scales, including your local machine, a local computing cluster, and on the cloud. </br>
 
-Cromwell on Azure configures all Azure resources needed to run workflows through Cromwell on the Azure cloud, and uses the [GA4GH TES](https://cromwell.readthedocs.io/en/develop/backends/TES/) backend for orchestrating the tasks that create a workflow. The installation sets up a VM host to run the Cromwell server and uses Azure Batch to spin up virtual machines that run each task in a workflow. Cromwell on Azure supports workflows written in the [Workflow Description Language or WDL](https://openwdl.org/) format.<br/>
+Cromwell on Azure configures all Azure resources needed to run workflows through Cromwell on the Azure cloud, and uses the [GA4GH TES](https://cromwell.readthedocs.io/en/develop/backends/TES/) backend for orchestrating the tasks that create a workflow. The installation sets up a VM host to run the Cromwell server and uses Azure Batch to spin up virtual machines that run each task in a workflow. Cromwell on Azure supports workflows written in the [Workflow Description Language or WDL](https://openwdl.org/) format. Cromwell workflows are written using the open source [WDL](https://github.com/openwdl/wdl) or the [CWL](https://www.commonwl.org/) scripting languages. To see examples of WDL scripts - see this ['Learn WDL'](https://github.com/openwdl/learn-wdl) repository on GitHub.<br/>

```

Comment:
Workflows can be written in WDL or CWL.

This is redundant: "Cromwell on Azure supports workflows written in the [Workflow Description Language or WDL](https://openwdl.org/) format. Cromwell workflows are written using the open source [WDL](https://github.com/openwdl/wdl) or the [CWL](https://www.commonwl.org/) scripting languages."

Also please add a link to CWL examples, if we are going to provide a link to WDL examples. 

WDL and CWL should be treated equally in these documents.

Thank you